From ba1da04104c97ed28ac12ce9b4bee595635ede97 Mon Sep 17 00:00:00 2001
From: Satyansh Shukla <satyansh.shukla@oracle.com>
Date: Mon, 6 Jul 2026 06:33:57 -0600
Subject: [PATCH 03708/15333] mmc: sdhci-of-dwcmshc: Log eMMC reset calls

Log when the BlueField-3 eMMC hardware reset path is invoked.

This makes it easier to diagnose cases where the controller recovery path
depends on issuing an eMMC reset, and helps confirm that the reset sequence
was attempted on affected systems.

Signed-off-by: Satyansh Shukla <satyansh.shukla@oracle.com>
Signed-off-by: Ulf Hansson <ulfh@kernel.org>
---
 drivers/mmc/host/sdhci-of-dwcmshc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/sdhci-of-dwcmshc.c b/drivers/mmc/host/sdhci-of-dwcmshc.c
index ba942c76f9ca..1baa2379c9c3 100644
--- a/drivers/mmc/host/sdhci-of-dwcmshc.c
+++ b/drivers/mmc/host/sdhci-of-dwcmshc.c
@@ -2013,6 +2013,7 @@ static void dwcmshc_bf3_hw_reset(struct sdhci_host *host)
 {
 	struct arm_smccc_res res = { 0 };
 
+	pr_debug("%s: resetting...\n", __func__);
 	arm_smccc_smc(BLUEFIELD_SMC_SET_EMMC_RST_N, 0, 0, 0, 0, 0, 0, 0, &res);
 
 	if (res.a0)
-- 
2.55.0

